src/rank_llm/retrieve_and_rerank.py

```diff
@@ -36,9 +39,16 @@ def retrieve_and_rerank(
     index_path: str = None,
     topics_path: str = None,
     index_type: str = None,
+    interactive: bool = False,
+    host: str = "http://localhost:8081",
+    exec_summary: bool = False,

```

Comment:
please rename `exec_summary` to `populate_exec_summary` here and everywhere else to better align with the intended boolean value of the arg.